README.rst

```diff
@@ -19,7 +19,7 @@ protocol on non-blocking sockets. The API derives from the original PyMongo.
 
 Compatibility
 -------------
-Python 2.7, 3.0, 3.3, 3.4 and PyPy.
+Python 2.7, 3.0, 3.3, 3.4, 3.5 and PyPy.
```

Comment:
I believe we can safely drop 3.0 and leave only "*Python 2.7, 3.3+ and PyPy*" here. Nobody seems to support Python 3.0–3.2, including Twisted.